schedview/collect/visits.py

```diff
@@ -65,6 +65,9 @@ def read_ddf_visits(*args, **kwargs) -> pd.DataFrame:
     all_visits = read_visits(*args, **kwargs)
 
     ddf_field_names = tuple(ddf_locations().keys())
+    # Different versions of the schedule include a DD: prefix, or not.
+    # Catch them all.
+    ddf_field_names = ddf_field_names + tuple([f"DD:{n}" for n in ddf_field_names])
```

Comment:
In new baselines, the Roman field shows up as "DD: RGES" .. although it's not really a DDF.
This is probably something to fix in the rubin_scheduler outputs, but it's worth noting here too. 
Creating similar kinds of plots for the Roman field is likely not terrible.